src/screens/NewTerm.js

```diff
@@ -0,0 +1,201 @@
+                    navigation.navigate('Home');
+                  }}>
+                  <BackIcon width={24} height={24} fill="#FFFFFF" />
+                </ToolButton>
+              }
+            />
+            <ToolBar
+              button={
+                <LanguageSwitcher
+                  firstLanguage={LANGUAGE_NAMES.EN}
+                  LeftText="EN"
+                  RightText="MA"
+                  color="#FFFFFF"
+                  iconType=""
+                  iconName="swap-horiz"
+                  onPress={() => null}
+                  iconSize={24}
+                />
+              }
+            />
+            <ToolBar
+              button={
+                <ToolButton onPress={action('clicked-add-button')}>
+                  <ModeIcon width={24} height={24} fill="#FFFFFF" />
+                </ToolButton>
+              }
+            />
+          </View>
+          <View style={styles.headerCategoryWrapper}>
+            <SectionHeading text="Category: " />
+            <View>
+              <Picker
+                selectedValue={selectedCategory}
+                style={{
+                  width: 300,
+                  height: 100,
+                }}
+                mode="dropdown"
+                onValueChange={Text => findSelectedId(Text)}>
+                <Picker.Item
+                  label={'Select Category'}
+                  value={null}
+                  color="#06B6D4"
+                />
+                {categories.map(categoryName => (
+                  <Picker.Item
+                    label={categoryName.name.en}
+                    key={categoryName.id}
+                    value={categoryName.name.en}
+                  />
+                ))}
+              </Picker>
+            </View>
+          </View>
+          <View style={styles.headerPhrases}>
+            <Text style={{paddingBottom: 15}}>
+              <SectionHeading text="The phrase in English: " />
+            </Text>
+            <Textarea
+              phrase={englishText}
+              placeholder="Enter here"
+              editable={true}
+              multiline={true}
+              onChange={Text => setEnglishText(Text)}
+            />
+          </View>
+          <View style={styles.headerPhrases}>
+            <Text style={{paddingBottom: 15}}>
+              <SectionHeading text="The phrase in Malagasy: " />
+            </Text>
+            <Textarea
+              placeholder="Enter here"
+              editable={true}
+              multiline={true}
+              phrase={malagasyText}
+              onChange={Text => setMalagasyText(Text)}
+            />
+          </View>
+          <View style={styles.addButton}>
+            <NextButton
+              onPress={() => {
+                addnewPhraseForCategory();
+              }}
+              text="Add"
+              value={englishText}
+              textColor={
+                malagasyText !== '' &&
```

Comment:
Consider refactoring this as you repeat it. An empty string is nullish
```suggestion
                malagasyText &&
```